library/ws-widget/collection/src/lib/_services/widget-content.service.ts

```diff
@@ -63,6 +63,27 @@ export class WidgetContentService {
   //     .pipe(retry(1))
   // }
 
+  fetchHierarchyContent(contentId: string): Observable<NsContent.IContent> {
```

Comment:
put api url in const const API_END_POINTS.

Reply:
Yes, will add it. Thanks for pointing it out.